roles/satellite-clone/tasks/main.yml

```diff
@@ -250,16 +259,8 @@
   - name: Wait for foreman-tasks service to start
     command: sleep 300
 
-  - name: Get candlepin credentials
-    command: sed -n -e 's/^.*myDS.password=//p' /etc/candlepin/candlepin.conf
-    register: candlepin_password
-
-  - set_fact:
-      clone_candlepin_password: "{{ candlepin_password.stdout }}"
-      cacheable: true
```

Comment:
This 'caching facts' thing was to try to make it so sat-clone could start at a step after a failed or incomplete run and still have all the context, but it really doesn't work well in practice and I haven't seen anyone use it. I don't see any value in keeping it, ideally we remove the whole feature.